src/components/ReservationsCard.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div class="reservations-card">
+    <div class="reservations-card__wrapper-first">
+      <h2 class="reservations-card__title">Upcoming</h2>
+      <BookingConfirmationCard
+        class="reservations-card__card"
+        v-for="reservation in reservations"
+        :key="reservation.id"
+        :id="reservation.id"
+        :title="reservation.title"
+        :seat="reservation.seat"
+        :datetime="reservation.datetime"
+        :ticketType="reservation.ticketType"
+        :reservationStatus="reservation.reservationStatus"
```

Comment:
Considering we pass so many props, but all coming from the `reservation` object - how about we pass the whole reservation object only? 🤔